tests/test_binding.py

```diff
@@ -6,19 +6,26 @@
 
 BINDING1 = Binding("a,b", action="action1", description="description1")
 BINDING2 = Binding("c", action="action2", description="description2")
+BINDING3 = Binding(" d   , e ", action="action3", description="description3")
```

Comment:
Is there a motivating use-case for binding keys that have surrounding whitespace?

Although I get that it provides more leeway for developers ([Postel's Law](https://en.wikipedia.org/wiki/Robustness_principle), perhaps), there could be a lot said in support of keeping the binding format concise (would unique-representation-per-key be achievable?).

Reply:
Being more robust was my motivation. Space is bound as the name of the key, not the printable character, and comma obviously has to be bound by name too. It's likely some folk might write those binding lists similar to how they'd write a list anyway ("a, b, c, d") so it seemed to make sense to accept that, given there was zero cost in doing so.